patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -9,27 +9,29 @@
 +    protected java.util.HashMap<Integer, Boolean> spawnForcedMap = new java.util.HashMap<Integer, Boolean>();
 +    public float eyeHeight = this.getDefaultEyeHeight();
 +    public static final net.minecraft.entity.ai.attributes.IAttribute REACH_DISTANCE = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "generic.reachDistance", 5.0D, 0.0D, 1024.0D).func_111112_a(true);
++    public static final net.minecraft.entity.ai.attributes.IAttribute ELYTRA_FLIGHT = new net.minecraft.entity.ai.attributes.RangedAttribute(null, "forge.elytraFlight", 0.0D, 0.0D, 1.0D).func_111112_a(true);
```

Comment:
Do you mean the "forge.elytraFlight"? If so, then that's because `EntityLivingBase` has a swim speed attribute that's named "forge.swimSpeed", which is what I based it on. I don't know which format is preferred because they're both present and available.